ui/hooks/useCarouselManagement/useCarouselManagement.ts

```diff
@@ -129,8 +132,7 @@ export const useCarouselManagement = ({
     }
     // Handle Contentful Data
     const maybeFetchContentful = async () => {
-      const contentfulEnabled =
-        remoteFeatureFlags?.contentfulCarouselEnabled ?? false;
+      const contentfulEnabled = false;
```

Comment:
Any reason not to remove the content that is no longer accessible after setting this to `false`?

Reply:
LOOOL good catch! I was disabling this for debugging purposes.
Will revert

Reply:
https://github.com/MetaMask/metamask-extension/pull/33243/commits/a75f8f09f06015dbc90662a980d2a1f583290f70

Reply:
☠️